src/applications/claims-status/reducers/claim-sync.js

```diff
@@ -17,13 +17,13 @@ export default function claimDetailReducer(state = initialState, action) {
   switch (action.type) {
     case SET_CLAIM_DETAIL:
       return _.assign(state, {
-        synced: action.meta.successfulSync,
+        synced: action.meta.syncStatus === 'SUCCESS',
         available: true,
         authorized: true
       });
     case SET_CLAIMS:
       return _.assign(state, {
-        synced: action.meta.successfulSync,
+        synced: action.meta.syncStatus === 'SUCCESS',
```

Comment:
The warning page (ClaimSyncWarning.jsx) is displayed if `synced` is false and the `synced` assignment had not been updated, along with the rest of the polling changes.